test/helpers/testing/jasmine/main.js

```diff
@@ -1,11 +1,10 @@
+//* eslint-env jasmine */
```

Comment:
```suggestion
/* eslint-env jasmine */
```

I've updated the `eslint-comments/no-use` configuration so `//*` is not needed anymore

Reply:
Ok, I will remove it, can you show me where update `eslint-comments/no-use`?

Reply:
I see it, I update source but forgot re-install so I see the error.